mutation/mutation_fragment.hh

```diff
@@ -112,6 +115,10 @@ public:
                 && _row.equal(column_kind::regular_column, s, other._row, s);
     }
 
+    int32_t merge_count() const {
+        return _merge_count;
```

Comment:
nit: unsigned?

Reply:
Yes, size_t would be better. Used int23_t though because there were already maps using this type in e.g. `system_keyspace::update_compaction_history`. This way I could focused first on covering all paths and leave this as the last step.

Reply:
size_t is useful for counting in-memory objects. Rows are on-disk, so uint64_t is indicated.

Reply:
Ah, that number corresponds to the number of sstables merged. That has to be a small number, it's not proportional to memory size.